java/src/test/java/com/saucelabs/simplesauce/EdgeTest.java

```diff
@@ -1,72 +1,63 @@
 package com.saucelabs.simplesauce;
 
 import org.junit.Test;
-import org.mockito.Mockito;
 
 import static org.junit.Assert.assertEquals;
 
 public class EdgeTest extends BaseConfigurationTest {
     @Test
     public void withEdge_default() {
         sauceOptions.withEdge();
-        startSauceSession();
 
-        String actualBrowserSetInConfig = sauce.getCurrentSessionCapabilities().getVersion();
+        String actualBrowserSetInConfig = sauceOptions.getBrowserVersion();
         assertEquals("18.17763", actualBrowserSetInConfig);
     }
 
     @Test
     public void withEdge18_returnsBrowserVersion18_17763() {
         sauceOptions.withEdge18();
```

Comment:
Same with all of these other tests, they seem to be only setter tests versus actual object creation tests

Reply:
I agree that we shouldn't be writing unit tests for lombok, but in this case we are adding logic to the setter, so this is where it gets unit tested.